src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -59,6 +59,8 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
    *                     if choosing to overwrite, only one backup will be maintained
    * @param excludes     which bronze targets to exclude from the snapshot
    */
+ //TODO: Add link for new Snapshot Class functionality
+  @deprecated("This Method is deprecated, Use new Snapshot Class instead. Please check the link for more details")
   def snapshot(
```

Comment:
This signature should not change but the code should ONLY call the new Spanshot.scala functions necessary to complete the same functionality. The process should work identically but this code should be removed so we don't have to "support" two snapshot processes while this is in deprecated state.